lib/types.ts

```diff
@@ -66,16 +67,18 @@ export const ACTIVITY_TYPES = [
   "pr_opened",
   "pr_merged",
   "pr_collaborated",
+  "discussion",
 ] as const;
 
 export interface Activity {
   type: (typeof ACTIVITY_TYPES)[number];
   title: string;
   time: string;
   link: string;
-  text: string;
+  text?: string;
```

Comment:
Could you specify why this was made nullable?

Reply:
Because there is no need of a text field while parsing GitHub discussion (in api.ts), However I will revert this and put
text : "", in GitHub discussion

Reply:
What about the discussion body? That can be kept as the text right?

Reply:
When you see GithubDicussion in GithubActivity.tsx there is no need to use text. 
To render the discussion body we already pass the whole discussion in <GithubDicussion.tsx >.